src/GafferCycles/IECoreCyclesPreview/CameraAlgo.cpp

```diff
@@ -115,27 +122,22 @@ ccl::Camera *convertCommon( const IECoreScene::Camera *camera, const std::string
 		ccam->border.clamp();
 	}
 	
-	// Shutter TODO: Need to see if this is correct or not, cycles also has a shutter curve...
+	// Shutter TODO: Cycles also has a shutter curve...
 	const Imath::V2f &shutter = camera->getShutter();
-	if ((shutter.x > 0.0) && (shutter.y > 0.0))
+	ccam->shuttertime = shutter.y - shutter.x;
```

Comment:
I'm thinking we might need to add an `abs` here eg. `abs( shutter.y - shutter.x );` if we render with a negative frame.